app/views/pages/device-analytics.html.erb

```diff
@@ -7,14 +7,14 @@
 	</li>
 </ul>
 <div class="row">
-	<div class="col-12 my-4">
+	<div class="col-12 mt-4">
```

Comment:
file names should be snake cased which means `underline` and not `dash`
snake_case_words
camelCaseWords
kebab-case-words, caterpillar-case-words, dash-case-words, and hyphen-case-words ( many terms for hyphenated words)